StudentAppMvc/StudentAppMvc/Exceptions/ObjectExistInDTOException.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace StudentAppMvc.Exceptions
+{
+    public class ObjectExistInDTOException : Exception
```

Comment:
Good to see customized exception. Perhaps, could be good if we custom a little bit message so that we can see our result :)